src/features/video/videoSlice.js

```diff
@@ -0,0 +1,44 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+
+const initialState ={
+    videos:[],
+    videoLoader:false,
+}
+
+export const getVideoList = createAsyncThunk('videoList/getVideoList',async (_,{rejectWithValue})=>{
+    try {
+        const res = await axios.get("/api/videos");
```

Comment:
Can write it as 
```
const {data:{videos}}=await axios.get("/api/videos")
return videos 
```
Can help in accessing nested values

Reply:
Sure, will do it